modules/migration-mtc-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-6_{context}"]
+= {mtc-full} 1.8.6 release notes
+
+[id="technical-changes-1-8-6_{context}"]
+== Technical changes
+
+.Multiple migration plans for a namespace is not supported
+
+{mtc-short} version 1.8.6 and later do not support multiple migration plans for a single namespace.
+
+.VM storage migration
+
+VM storage migration feature changes status from Technical Preview status to being generally available.
```

Comment:
```suggestion
VM storage migration feature changes from Technical Preview status to being Generally Available.
```
Should "Generally Available" here be capitalized similar to "Technical Preview"?
Remove one of the use of 'status' here?